gator/run.py

```diff
@@ -31,7 +31,7 @@ def get_actual_output(output):
         try:
             current_line_decoded = line.decode()
         # line cannot decode, return the line itself
-        except AttributeError:
+        except ValueError:
```

Comment:
I think that this should be kept (it tests for an error case that was not tested before), but the `AttributeError` `except` needs to stay as well. A test should be generated to test it, possibly by passing in to `get_actual_output` an `str` object.